tests/bundle/DependencyInjection/Configuration/Parser/AdminUiFormsTest.php

```diff
@@ -116,6 +116,47 @@ public function testContentEditFieldTypesAreMapped(): void
 
         $this->parser->mapConfig($scopeSettings, $currentScope, $this->contextualizer);
     }
+
+    /**
+     * Test 'meta_field_groups_list' fieldtype settings are mapped.
+     */
+    public function testContentEditMetaFieldgroupListIsMapped(): void
+    {
+        $scopeSettings = [
+            'admin_ui_forms' => [
+                'content_edit' => [
+                    'meta_field_groups_list' => [
+                        'metadata',
+                        'seo',
+                    ],
+                ],
+            ],
+        ];
+        $currentScope = 'admin_group';
+
+        $this->contextualizer
+            ->expects($this->atLeast(2))
```

Comment:
We have preference to use static methods in PHPUnit by using static notation.

```suggestion
            ->expects(self::atLeast(2))
```